api/numaresourcesoperator/v1/numaresourcesoperator_defaults.go

```diff
@@ -20,6 +20,7 @@ import (
 	"time"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
```

Comment:
I prefer to change all the instances in one go (to be done in a follow up PR) rather than add just for this change

Reply:
not sure I understand, what instances are you referring to? 
in another comment I see you do not like boolean so will make that a string with 2 options: enable & disable. that way we won't need this lib

Reply:
there are other places in the codebase which can benefit to move to `k8s.io/utils/pointer`. I actually welcome this change. But it should be done all in once: the only thing I'm not fine with is having both the old clunky way I started (reinventing `k8s.io/utils/pointer` locally) and using `k8s.io/utils/pointer`.  Either you submit a PR to convert to `k8s.io/utils/pointer` everywhere and get that PR merged before to move on with this one (fine) or you use the old way in this new PR and convert to `k8s.io/utils/pointer` with a later PR

Reply:
okay makes sense, thanks for clarifying. I will handle this in a later PR.